CommandLine/protos/server.proto

```diff
@@ -86,3 +87,8 @@ message SystemInfo {
   optional string author = 4; ///< Name of the system author or maintainer (e.g, "John Doe")
   optional string target = 5; ///< The target-platform of the system. Only used by compiler descriptions presently.
 }
+
+message OutputReply {
+  optional string out = 1; ///< All available output read from the standard output stream.
```

Comment:
This is a C way of thinking. Make this a streaming RPC of compiler message. Insoide Google, we use `FATAL`, `ERROR`, `WARN`, `INFO`, `FINE`, `FINER`, `FINEST`. I'd recommend taking `ERROR` .. `FINE`. Eventually, we'll use a logger similar to Google's to surface messages to the user, and your IDE can sort them and display them accordingly.

So, consider this:
```proto
message LogMessage {
  enum Severity {
    UNKNOWN = 0,
    FINE = 1,
    INFO = 2,
    WARNING = 3,
    ERROR = 4,
  }
  optional Severity severity = 1;
  optional string message = 2;
}
```

Reply:
Sure: 36dac03f55f24fc0d46082042cfd1de594e6d234

I wasn't too fascinated with the idea of message logging, all I care about personally is basic text, but yeah. We've been saying we need better logging for a while.